product_docs/docs/pgd/5/cli/command_ref/pgd_switchover.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: switchover
+---
+
+Switchover to new write leader.
+
+### Synopsis
+
+Switchover to new write leader. Use switchover method fast for immediate
+switchover, or strict to wait until lag is less than route_writer_max_lag
+on given target node. If switchover fails due to timeout or any other
+issue, BDR may elect a write leader which is different than given target
+node.
+
```

Comment:
Revise to:
on the given target node. If switchover fails due to timeout or any other
issue, BDR might elect a write leader that's different from the given target
node.

Reply:
fixed